src/scripts/download-new-songs.ts

```diff
@@ -133,17 +133,88 @@ const downloadNewSongs = async (limit?: number) => {
         await db.kmq("not_downloaded").insert(songIdsNotDownloaded).transacting(trx);
     })
     await db.destroy();
-    logger.info(`Total songs downloaded: ${downloadCount}`);
+    logger.info(`Total songs downloaded: ${downloadCount}, (${deadLinksSkipped} dead links skipped)`);
+}
+
+const convertToOpus = async (files: Array<string>) => {
+    const endingWithMp3Regex = new RegExp("\\.mp3$");
+    const mp3Files = files.filter((file) => file.match(endingWithMp3Regex));
+    logger.info(`Converting ${mp3Files.length} from mp3 to opus (in ogg container)`)
+
+    ffmpegOpusJob(mp3Files);
+    logger.info("Completed jobs of length", mp3Files.length);
+}
+
+const ffmpegOpusJob = (files: Array<string>) => {
+    files.forEach(async (mp3File) => {
+        try {
+            let oggFileWithPath: string = path.join(process.env.SONG_DOWNLOAD_DIR, `${path.basename(mp3File, ".mp3")}.ogg`);
+            if (fs.existsSync(oggFileWithPath)) {
+                return;
```

Comment:
Should this be `continue;`?

Reply:
I just checked, the equivalent for `continue` in a `forEach` is `return`